deployment/docker/pretalx.bash

```diff
@@ -1,23 +1,50 @@
 #!/bin/bash
-cd /pretalx/src
-export PRETALX_DATA_DIR=/data
+cd /pretalx/src || exit
```

Comment:
Shouldn't this at least exit with an error? As-is, users would just see a successful exit while nothing happens.

Reply:
`cd` itself prints an error message if the working directory could not be changed. `exit` seems to return a non-zero status (or possibly `$?`?) if none is explicitly specified, but since I am not sure this is always the case, I have changed it to `exit 1`.